### PR TITLE
P0 login signup pages

### PR DESCRIPTION
**Pull request by eldhoabraham94**
Adding login signup pages

### CODE REVIEW

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  font-size: 1.1rem;
+}
+
+.category-container {
+  list-style: none;
+  margin-left: 0px;
+  font-size: 14px;
+  padding:0.4rem;
+}
+
```
Comment (deekshithmd) [inline product/product.css:36]: avoid unnecessary line gaps, try to give a line gap between classes

**Review by deekshithmd — APPROVED** [review]

File: login/login.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                            <div class="dropdown-content">
+                                <div class="dropdown-items">
+                                    <a href="#">Orders</a>
+                                    <a href="#">Wishlist</a>
+                                </div>
+
+                            </div>
+                        </div>
+                        <div class="icon-ecomm">
+                            <a href="/WishList/Wishlist.html"><i class="far fa-heart fa-lg"></i></a>
```
Comment (swethaogeti) [inline login/login.html:36]: clean code ! just reset the code

File: product/product.html [hunk 3]
```diff
@@ -0,0 +1,298 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (swethaogeti) [inline product/product.html:8]: change title

File: login/login.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+.login-form input{
+    display: block;
+}
+.login-form div{
+    padding:10px;
+}
+
+.form-input-container{
+    width: 300px;
```
Comment (swethaogeti) [inline login/login.css:23]: try using rem

File: login/login.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              </div>
+              
+              <a href="#" class="button-login-form border-style">Login</a>
+              <a class="button-link" href="signup.html">Create New Account <i class="fas fa-chevron-right"></i></a>
+            </form>
+          </div>
+
+    </div>
+
+
```
Comment (swethaogeti) [inline login/login.html:65]: remove extra spacing

**Review by swethaogeti — APPROVED** [review]
clean code !! can merge

**Pull request merged**